packages/boxel-ui/addon/components/modal/index.gts

```diff
@@ -58,6 +58,7 @@ export default class Modal extends Component<Signature> {
             small=(eq @size 'small')
             medium=(eq @size 'medium')
             large=(eq @size 'large')
+            full-screen=(eq @size 'full-screen')
           }}
```

Comment:
Any reason this can't be `<dialog class={{@size}} ... />`?

Reply:
good point!